packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.tsx

```diff
@@ -0,0 +1,247 @@
+                      <span className={classes.denominator}>
+                        {convertToLocaleString(props.ratio[1], props.culture)}
+                      </span>
+                    </>
+                  </div>
+                )}
+              </div>
+              {!!props.descriptionMessage && (
+                <div className={classes.descriptionMessage}>{props.descriptionMessage}</div>
+              )}
+            </div>
+          )}
+        </PopoverSurface>
+      </Popover>
+    </div>
+  );
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function _multiValueCallout() {
+    const yValueHoverSubCountsExists: boolean = _yValueHoverSubCountsExists(props.YValueHover);
+    return (
+      <div className={classes.calloutContentRoot}>
+        <div
+          className={classes.calloutDateTimeContainer}
+          style={yValueHoverSubCountsExists ? { marginBottom: '11px' } : {}}
+        >
+          <div
+            className={classes.calloutContentX}
+            {...getAccessibleDataObject(props!.xAxisCalloutAccessibilityData, 'text', false)}
+          >
+            {convertToLocaleString(props!.hoverXValue, props.culture)}
+          </div>
+        </div>
+        <div style={yValueHoverSubCountsExists ? { display: 'flex' } : {}}>
+          {props!.YValueHover &&
+            props!.YValueHover.map((yValue: YValueHover, index: number, yValues: YValueHover[]) => {
+              const isLast: boolean = index + 1 === yValues.length;
+              const { shouldDrawBorderBottom = false } = yValue;
+              return (
+                <div
+                  {...getAccessibleDataObject(yValue.callOutAccessibilityData, 'text', false)}
+                  key={`callout-content-${index}`}
+                  style={
+                    yValueHoverSubCountsExists
+                      ? {
+                          display: 'inline-block',
+                          ...(shouldDrawBorderBottom && {
+                            borderBottom: `1px solid ${tokens.colorNeutralStroke2}`,
+                            paddingBottom: '10px',
+                          }),
+                        }
+                      : {
+                          ...(shouldDrawBorderBottom && {
+                            borderBottom: `1px solid ${tokens.colorNeutralStroke2}`,
+                            paddingBottom: '10px',
+                          }),
+                        }
+                  }
+                >
+                  {_getCalloutContent(yValue, index, yValueHoverSubCountsExists, isLast)}
+                </div>
+              );
+            })}
+          {!!props.descriptionMessage && <div className={classes.descriptionMessage}>{props.descriptionMessage}</div>}
+        </div>
+      </div>
+    );
+  }
+
+  function _yValueHoverSubCountsExists(yValueHover?: YValueHover[]) {
+    if (yValueHover) {
+      return yValueHover.some(
```

Comment:
Nit: You can return yValueHover && yValueHover.some( ... to avoid the if statement and return false statement.

Reply:
@srmukher Could you address this comment

Reply:
done